manual/manual/html_processing/images/colour-logo-gray.svg

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I don't think that we should commit any images in the tree: for local build, they should be downloaded from the ocaml.org source in the makefile, and once deployed to ocaml.org, the manual should use the resource here. 

There is also an issue with the favicon not appearing in the manual part, and only in the API part of the documentation.

Commenting on this favicon, there are few  more issues:
- there is`#` overlaid on the usual logo
- it is far too big, 130K compared to the 1K favicon served by ocaml.org
It seems really better to get back to `ocaml.org` favicon here.

Reply:
there was a discussion about the favicon here: https://github.com/sanette/ocaml-api/issues/2

Reply:
Having a distinct favicon for the experimental or local version is an interesting idea. However, this argument does not seems to apply for the production version. (And if we have a local version of the favicon, I would vote for a svg one since this means text data rather than binary).

Reply:
> I don't think that we should commit any images in the tree: for local build, they should be downloaded from the ocaml.org source in the makefile, and once deployed to ocaml.org, the manual should use the resource here.

OK, then I've openned this PR: https://github.com/ocaml/ocaml.org/pull/1166
For the search icon, I will use https://ocaml.org/img/search.svg